### ISSUE TITLE
Whither angle bracket?

### ISSUE THREAD

**Issue by lfrancis** [Threespot/aldf-wp#92]
http://live-aldf-shop.pantheonsite.io/product/aldf-womens-t-shirt/

I'm guesisng that angle bracket after _description_ isn't supposed to off floating in space on its own?

![angled](https://user-images.githubusercontent.com/6942589/40324989-47ec4d84-5d08-11e8-9205-80f4b7658e58.png)

**Comment by lfhello** [comment]
@lfrancis I don't know if we need to make a new ticket in the Shop repo but her eis a link https://github.com/Threespot/aldf-shop

And @Henryhankdc if we can move that arrow to right after Description, that'd be great- thanks!

**Comment by Henryhankdc** [comment]
@lfrancis and @lfhello Although the angle bracket looks sad and lonely hanging out there it does server a purpose. In this theme it points to the corresponding content displayed when the user clicks one of these options. Whichever options is currently selected is where the bracket will appear. The built in style is to be placed at the end of each row. Let me know if you would like me to try to customize the bracket to appear next to each item instead of at the end of the row.

**Comment by lfhello** [comment]
@Henryhankdc I don't mind it over there- I remember what it's pointing to and I think users will still know how to interact with it. If it's a bigger lift to adjust that, I can live with it there.

**Comment by lfrancis** [comment]
So is the consensus to leave it where it is and not move it to the end of the description copy?

**Comment by lfhello** [comment]
Yes, I think we can keep it where it is @lfrancis  @Henryhankdc . 

It does directly point to the area where the information is shown. 

If it were taking you to another page, I would want it moved closer to Description, but since it is just revealing information on the same page, it's actually easier to see what is changing.

![screen shot 2018-05-31 at 10 49 58 am](https://user-images.githubusercontent.com/17705791/40799415-88885f10-64db-11e8-8bbb-ed9db7178321.png)
